business/istio_config.go

```diff
@@ -1230,6 +1229,11 @@ func (in *IstioConfigService) GetIstioConfigPermissions(ctx context.Context, nam
 
 	istioConfigPermissions := make(models.IstioConfigPermissions, len(namespaces))
 
+	_, ok := in.userClients[cluster]
+	if !ok {
+		cluster = in.config.KubernetesConfig.ClusterName
```

Comment:
can you put a comment here why the home cluster is used for permissions checks when provided cluster does not exist?

Reply:
Thanks for pointing at this, @hhovsepy . I think it should return error in this case.